common/unsafe/src/test/scala/org/apache/spark/unsafe/types/CollationFactorySuite.scala

```diff
@@ -369,8 +369,6 @@ class CollationFactorySuite extends AnyFunSuite with Matchers { // scalastyle:ig
       1 << 15, // UTF8_BINARY mandatory zero bit 15 breach.
       1 << 16, // UTF8_BINARY mandatory zero bit 16 breach.
       1 << 17, // UTF8_BINARY mandatory zero bit 17 breach.
-      1 << 18, // UTF8_BINARY mandatory zero bit 18 breach.
-      1 << 19, // UTF8_BINARY mandatory zero bit 19 breach.
```

Comment:
and what about bit 20? do we still keep it as mandatory zero
also, what about bit 21? I don't even see that one here...

Reply:
Yes we do, untill we find a reason to use it, it should be reserved and fixed to 0.
There were no test for 21, will add it.